src/DynamoPublish/ViewModels/PublishViewModel.cs

```diff
@@ -130,6 +135,8 @@ public string ManagerURL
         #region Click commands
 
         public DelegateCommand PublishCommand { get; private set; }
+        public DelegateCommand VisitCommand { get; private set; }
+        public DelegateCommand CopyLinkCommand { get; private set; }
```

Comment:
One new button is added to the UI to visit a customizer. The copy link button now uses a command rather than a callback in the code-behind. This means it is properly disabled when appropriate.